lib/tests.nix

```diff
@@ -278,3 +248,45 @@ runTests {
   };
 
 }
+
+
+# MISC
+
+  testOverridableDelayableArgsTest = {
```

Comment:
Good catch, this looked like a GH formatting hickup to me, but I had to merge a syntax change in the function to get to master again and screwed up the positioning.